src/test/shared/cloudformation/templateRegistry.test.ts

```diff
@@ -339,6 +340,202 @@ describe('CloudFormation Template Registry', async () => {
             ])
         })
     })
+
+    describe('getResourcesForHandlerFromTemplateDatum (image tests)', async () => {
+        let tempFolder: string
+        let helloPath: string
+        let nestedPath: string
+
+        beforeEach(async () => {
+            tempFolder = await makeTemporaryToolkitFolder()
+            helloPath = path.join(tempFolder, 'hello-world')
+            nestedPath = path.join(helloPath, 'nested')
+            await fs.mkdirp(nestedPath)
+            await fs.mkdirp(helloPath)
```

Comment:
`testUtils.toFile()` would avoid the need for these explicit mkdir's (maybe it should be in filesystemUtils instead)